### PR TITLE
LESS directory for CSS changes

### PR DESCRIPTION
**Pull request by roderickmorales**
This PR includes all the LESS files used for the compile CSS.

### CODE REVIEW

File: ab_tool/templates/ab_tool/edit_experiment.html [hunk 1]
```diff
@@ -254,6 +261,11 @@ <h4 class="modal-title">Edit Summary</h4>
     {% endif %}
     
     window.parentPage = "{% url 'ab_testing_tool_index' %}";
+    //error-only-numbers
+    $(document).ready(function(){
```
Comment (elliottyates) [inline ab_tool/templates/ab_tool/edit_experiment.html]: This should probably be removed -- looks like testing code.

File: ab_tool/templates/ab_tool/edit_experiment.html [hunk 2]
```diff
@@ -61,51 +61,58 @@
...
+                                        title="Edit {{track.name}}" ng-cloak
+                                        ng-click="editTrackName(track)" ng-if="!track.editing">
+                                        Track "{{track.name}}"
+                                    </a>
+                                    <label ng-if="track.editing">Track
+                                        <input data-track="name" title="Edit track name"
+                                            ng-cloak ng-model="track.newName"
+                                            ng-keypress="trackNameKeypress($event, track)"
+                                            ng-blur="trackNameChanged(track)">
+                                        <!--TODO:button>save</button>
```
Comment (elliottyates) [inline ab_tool/templates/ab_tool/edit_experiment.html]: Do we still need this?

File: ab_tool/templates/ab_tool/base_lti.html [hunk 3]
```diff
@@ -6,7 +6,6 @@
 
   <link href="{% static "favicon.ico" %}" rel="icon" type="image/x-icon">
   {% block stylesheet %}
-    <!--link rel="stylesheet" href="{% static 'ab_tool/css/bootstrap.min.css' %}" type="text/css"/-->
```
Comment (elliottyates) [inline ab_tool/templates/ab_tool/base_lti.html]: Can we remove this line entirely?

File: ab_tool/static/ab_tool/LESS/style.less [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+	left: 50%;
+	margin-left: -125px;
+	-o-box-shadow: 0 0 8px #111;
+	-moz-box-shadow: 0 0 8px #111;
+	-webkit-box-shadow: 0 0 8px #111;
+	box-shadow: 0 0 8px #111;
+	z-index: 10000;
+}
+
+/*#expSummary{
```
Comment (elliottyates) [inline ab_tool/static/ab_tool/LESS/style.less]: Do we still need this commented section?

**Comment by elliottyates** [comment]
Further questions:
1. There is a chance that the LESS and the CSS will get out-of-sync if the CSS is included in the project. We'll need some reminder / process for regenerating the CSS before checking in any changes to the LESS....?
2. `.fa-question-circle` isn't being used anymore in the IP modal forms, can we remove all that LESS code?
3. Any reason why form-lti and form LESS files can't reuse some code? Seems like a lot of duplication.

**Comment by roderickmorales** [comment]
I removed the <name>-lti.less files from the LESS directory. The  .fa-question-circle is for future functionality, so it will be nice to keep it there as a reminder

**Comment by roderickmorales** [comment]
Inclusion of the LESS directory.
Error handling for creating/editing experiment when the user gives a non-numeric value to a track weight.

**Pull request merged**